ci-operator/config/openshift-service-mesh/sail-operator/openshift-service-mesh-sail-operator-main.yaml

```diff
@@ -402,7 +419,10 @@ tests:
            git config --global user.name 'Jonh Wendell'; \
            cd /work && \
            export BUILD_WITH_CONTAINER="0" && \
+           export VERSIONS_YAML_FILE=versions.yaml && \
```

Comment:
this change is not required, we already remove the entire vendor directory and Makefile for the nightlies (as they're supposed to be community nightlies)

Reply:
Right, it's there just for clarity and in case somebody will decide to remove `rm Makefile.vendor.mk` in the future and use just env variables. Should I still remove it or it's ok to keep it?

Reply:
hmm. I guess it can stay, does no harm